tasks/gen.go

```diff
@@ -138,6 +139,18 @@ func main() {
 		schema.SpawnStructField("WorkedBy", "String", false, false),
 	}, schema.SpawnStructRepresentationMap(map[string]string{})))
 
+	// reputation api
+	ts.Accumulate(schema.SpawnStruct("AuthenticatedRecord", []schema.StructField{
+		schema.SpawnStructField("Record", "Link_FinishedTask", false, false),
+		schema.SpawnStructField("Signature", "Bytes", false, false),
+	}, schema.SpawnStructRepresentationMap(map[string]string{})))
+	ts.Accumulate(schema.SpawnList("List_AuthenticatedRecord", "AuthenticatedRecord", false))
+	ts.Accumulate(schema.SpawnStruct("RecordUpdate", []schema.StructField{
+		schema.SpawnStructField("Records", "List_AuthenticatedRecord", false, false),
+		schema.SpawnStructField("SigPrev", "Bytes", false, false),
+		schema.SpawnStructField("Previous", "Link", false, false),
```

Comment:
Link to...? I imagine this could/shoudl be something like Link_RecordUpdate.

Reply:
this seems to be a bug in graphql code gen. If i have a cycle of links (struct includes link to itself) we end up with a circular definition.